src/styles/blocks/card.scss

```diff
@@ -0,0 +1,64 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  background-color: $background-color-special-white;
```

Comment:
The variable `$background-color-special-white` is used but not defined. Please define this variable to ensure the background color is applied correctly.